src/core/classification/qgsclassificationequalinterval.h

```diff
@@ -35,13 +35,14 @@ class CORE_EXPORT QgsClassificationEqualInterval : public QgsClassificationMetho
     QgsClassificationMethod *clone() const override;
     QIcon icon() const override;
 
+    bool valuesRequired() const override { return false; }
+
     static const QString METHOD_ID;
 
   private:
-    QList<double> calculateBreaks( double minimum, double maximum,
+    QList<double> calculateBreaks( double &minimum, double &maximum,
```

Comment:
Can it be const?

Reply:
nope.
The point is that the method might update min/max depending on config.
And the methods were returning classes with upper bound only, so the base class needs to add the minimum once again.
I have chosen to not modify the behavior of the methods (by return the lower bound too) as it might have causes unexpected issues in QGIS (they are used in other places).
So to make it const we have to modify this.